CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
+
+- Fix `containerd` configuration file.
+
+### Changed
+
+- Move `.internal.localRegistryCache` to `global.localRegistryCache` to publicly expose the settings.
```

Comment:
It should be moved to `$.Values.global.components.containerd.localRegistryCache` (`$.Values.global.components.containerd` already exists).

Reply:
Updated here: https://github.com/giantswarm/cluster/pull/178/commits/b517710f9b51dcba63a228d995b5cc1c4e9fce8c